app/controllers/admin_invoices_controller.rb

```diff
@@ -5,5 +5,6 @@ def index
 
   def show
     @invoice = Invoice.find(params[:id])
+    @items = @invoice.items
```

Comment:
feel free to call this method in your views instead of setting up an variable to pass through the controller. I think this is fine as well, but a good option for any controllers that start to get large.